src/sbvr-api/compile-auth.ts

```diff
@@ -0,0 +1,53 @@
+import type { Filter, PinejsClientCore, Resource } from 'pinejs-client-core';
+
+const asPermission = (compiledFilter: string): string => {
+	return compiledFilter
+		.split('$filter=')[1]
```

Comment:
TODO: more strict checks on this... probably regex match the entire filter to avoid possible issues in case somehow a $select went by or something of the sort